### PR TITLE
Laundry response

### PR DESCRIPTION
**Pull request by VinaiRachakonda**
Added functionality with laundry keyword. Would like to test.

### CODE REVIEW

**Comment by ajmichaels** [comment]
The JS changes look good.  Could you add your xml files to the .gitignore?

**Review by armic — CHANGES_REQUESTED** [review]
the JS changes look good; could you add your xml files to the .gitignore?

File: index.js [hunk 1]
```diff
@@ -57,34 +53,50 @@ app.post('/webhook', function(req, res) {
...
-      sendTextMessage(senderID, messageText);
+      case 'laundry':
+        $.getJSON('https://api.pennlabs.org/laundry/halls', function (response) {
+          let hallsArray = response['halls'];
+          for (let i = 0; i < hallsArray.length; i++) {
+            let dryers_available = hallsArray[i]['dryers_available'];
+            let name = hallsArray[i]['name'];
+            let washers_available = hallsArray['washers_available'];
+
+            let ret = name.concat(": There are ").concat(dryers_available).concat(" available dryers and ")
```
Comment (armic) [inline index.js]: const ret = `${name}: There are ${dryers_available} available dryers and ${washers_available} washers available!`;

**Comment by VinaiRachakonda** [comment]
I fixed it. Ready to merge?

**Pull request merged**